Describing what could be done to make a text index work. (#1648)
neo4j/neo4j-documentation: cypher/cypher-docs/src/docs/dev/query-tuning-indexes.adoc

|substring search| `CONTAINS`
|===

> In some cases it is not possible for the system to determine an expression to be of type string.

Reviewer: ```suggestion
In some cases, the system cannot determine an expression of a type string.
```

Author: That does not cover what this should express. What do you think about:

```suggestion
In some cases, the system cannot determine whether an expression is of type string.
```

Reviewer: Sounds good to me, yes.

[source, cypher, role="noheader"]
----
MATCH (n:Label) WHERE n.prop = $param
----

> In such cases the author of the query can modify the query to provide this information. Depending on how values which are not of type string should be treated, there are two options:

Reviewer: ```suggestion
Such queries can be modified to provide this information. 
Depending on how values that are not of type String should be treated, there are two options:
```

Author: I like that. I am just curious whether you would capitalise "string" in this context. Then we should probably also do it in the first case.

Reviewer: Aaah, that's a leftover. First, I capitalized all of them, but then I saw that in the example, it's represented as `"string"`, so I removed the capitalization, or I thought I had removed it. 😄
